include/algorithms/logistic_regression/logistic_regression_predict_types.h

```diff
@@ -61,9 +61,9 @@ enum Method
 */
 enum ResultToComputeId
 {
-    computeClassesLabels = 0x00000001ULL,
-    computeClassesProbabilities = 0x00000002ULL,
-    computeClassesLogProbabilities = 0x00000004ULL
+    computeClassesLabels = classifier::computeClassesLabels,
+    computeClassesProbabilities = classifier::computeClassesProbabilities,
```

Comment:
Sorry for too delay.
seems we have to change computeClassesLabels to computeClassLabels according to Andrey Nikolaev comment regarding the naming.
for probabilities and log probabilities it`s applicable also.

Reply:
fixed